docs/pages/router/reference/hooks.mdx

```diff
@@ -113,6 +113,33 @@ export default function Route() {
 }
 ```
 
+For nested layouts, you can pass a secondary argument, denoting the layout route, to access higher-order layouts. For example:
```

Comment:
```suggestion
When using nested layouts, you can access higher-order layouts by passing a secondary argument denoting the layout route. For example:
```

it was a bit difficult to read with too much pause (`,` usage). So I tried to make it sound a bit linear.